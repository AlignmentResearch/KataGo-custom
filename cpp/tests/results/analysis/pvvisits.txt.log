: Running with following config:
cudaUseFP16 = false
cudaUseNHWC = false
forDeterministicTesting = true
logAllRequests = true
logAllResponses = true
logFile = tests/results/analysis/pvvisits.txt.log
logSearchInfo = true
logTimeStamp = false
maxPlayouts = 10000
maxVisits = 100
nnCacheSizePowerOfTwo = 23
nnMaxBatchSize = 64
nnMutexPoolSizePowerOfTwo = 17
nnRandSeed = analysisTest
nnRandomize = false
numAnalysisThreads = 1
numSearchThreads = 1
openclUseFP16 = false
reportAnalysisWinratesAs = BLACK
rootSymmetryPruning = false
trtUseFP16 = false

: Analysis Engine starting...
: KataGo v1.12.3
: nnRandSeed0 = analysisTest
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 19 * 19 allowing smaller boards
: Cuda backend thread 0: Found GPU NVIDIA RTX A6000 memory 51050250240 compute capability major 8 minor 6 device 0
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded config configs/analysis_example.cfg and/or command-line and query overrides
: Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: Config override: cudaUseFP16 = false
: Config override: cudaUseNHWC = false
: Config override: forDeterministicTesting = true
: Config override: logAllRequests = true
: Config override: logAllResponses = true
: Config override: logDir = 
: Config override: logFile = tests/results/analysis/pvvisits.txt.log
: Config override: logSearchInfo = true
: Config override: logTimeStamp = false
: Config override: maxPlayouts = 10000
: Config override: maxVisits = 100
: Config override: nnRandSeed = analysisTest
: Config override: nnRandomize = false
: Config override: numAnalysisThreads = 1
: Config override: numSearchThreadsPerAnalysisThread = 1
: Config override: openclUseFP16 = false
: Config override: rootSymmetryPruning = false
: Config override: trtUseFP16 = false
: Analyzing up to 1 positions at at time in parallel
: Started, ready to begin handling requests
: Request: {"id":"pvvisittest","initialStones":[],"moves":[["B","D4"],["W","D5"],["B","C5"],["W","E4"],["B","D3"],["W","C6"],["B","E5"],["W","D6"]],"rules":"tromp-taylor","komi":9,"boardXSize":7,"boardYSize":7,"overrideSettings":{"maxVisits":1000},"includePVVisits":true}
: Request: {"id":"pvvisittest","initialStones":[],"moves":[["B","D4"],["W","D5"],["B","C5"],["W","E4"],["B","D3"],["W","E5"],["B","C6"],["W","E3"],["B","E2"],["W","F2"],["B","D2"],["W","C4"],["B","D6"],["W","B4"]],"rules":"chinese","komi":9,"boardXSize":7,"boardYSize":7,"overrideSettings":{"maxVisits":1000},"includePVVisits":true}
: MoveNum: 8 HASH: 68923BD112145D32565C85CE0EF9CB35
   A B C D E F G
 7 . . . . . . .
 6 . . O1O3. . .
 5 . . X O X2. .
 4 . . . X O . .
 3 . . . X . . .
 2 . . . . . . .
 1 . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi9
Root visits: 1000
New playouts: 1000
NN rows: 860
NN batches: 860
NN avg batch size: 1
PV: E3 F4 F5 F6 F3 C4 B5 B4 B3 B6 C3 A5 E6 E7 G4 A3 A2 A4 B1 F7
Tree:
: T   3.81c W  12.31c S  -4.91c ( +0.5 L  +0.5) N    1000  --  E3 F4 F5 F6 F3 C4 B5
---Black(^)---
E3  : T  11.27c W  18.27c S  -5.40c ( +0.8 L  +0.7) LCB   -0.29c P  3.98% WF 521.2 PSV     486 N     501  --  E3 F4 F5 F6 F3 C4 B5 B4
F5  : T   3.93c W  13.99c S  -4.02c ( +0.7 L  +0.6) LCB  -14.25c P 18.75% WF 261.2 PSV     145 N     260  --  F5 B5 C4 B4 B3 E3 D2 E2
F4  : T -11.11c W  -1.95c S  -6.07c ( -0.1 L  -0.1) LCB  -32.18c P 36.13% WF 127.8 PSV      95 N     146  --  F4 C4 C3 B5 B4 F5 E3 E6
E6  : T -22.27c W  -9.88c S  -5.10c ( -0.6 L  -0.8) LCB  -44.77c P 40.37% WF  71.1 PSV      72 N      89  --  E6 E3 E2 F2 B5 B6 D2 F5
B5  : T -87.06c W -72.12c S -14.94c ( -5.4 L  -5.0) LCB -437.06c P  0.44% WF   0.8 PSV       0 N       1  --  B5
B4  : T -101.20c W -82.91c S -18.30c ( -7.0 L  -5.8) LCB -451.20c P  0.08% WF   0.7 PSV       0 N       1  --  B4
B6  : T -83.49c W -69.80c S -13.69c ( -5.5 L  -3.4) LCB -433.49c P  0.07% WF   0.8 PSV       0 N       1  --  B6

: Response: {"id":"pvvisittest","isDuringSearch":false,"moveInfos":[{"lcb":0.550073188,"move":"E3","order":0,"prior":0.0397898182,"pv":["E3","F4","F5","F6","F3","C4","B5","B4","B3","B6","C3","A5","E6","E7","G4","A3"],"pvEdgeVisits":[501,341,331,295,262,167,116,102,36,14,10,9,13,12,10,9],"pvVisits":[501,341,331,295,262,167,116,102,36,14,10,15,13,12,10,9],"scoreLead":0.727054998,"scoreMean":0.727054998,"scoreSelfplay":0.767866577,"scoreStdev":6.63931026,"utility":0.112668614,"utilityLcb":-0.00286120816,"visits":501,"weight":486.19663749964667,"winrate":0.591333839},{"lcb":0.505056348,"move":"F5","order":1,"prior":0.18750371,"pv":["F5","B5","C4","B4","B3","E3","D2","E2","F2","B2","E1","C2","F3"],"pvEdgeVisits":[260,227,134,94,84,73,63,39,16,9,4,2,1],"pvVisits":[260,227,134,94,84,73,63,39,16,9,4,2,1],"scoreLead":0.576046395,"scoreMean":0.576046395,"scoreSelfplay":0.704911421,"scoreStdev":11.1433361,"utility":0.0392741566,"utilityLcb":-0.14247914,"visits":260,"weight":259.5575635529757,"winrate":0.569968239},{"lcb":0.415005721,"move":"F4","order":2,"prior":0.361343056,"pv":["F4","C4","C3","B5","B4","F5","E3","E6","C5","E4","B6","C4","E5"],"pvEdgeVisits":[146,121,108,103,78,32,19,16,6,5,3,2,1],"pvVisits":[146,121,108,103,78,32,19,16,6,5,3,2,1],"scoreLead":-0.119258103,"scoreMean":-0.119258103,"scoreSelfplay":-0.125941616,"scoreStdev":9.67756693,"utility":-0.111081208,"utilityLcb":-0.321783473,"visits":146,"weight":146.0,"winrate":0.49025653},{"lcb":0.370236931,"move":"E6","order":3,"prior":0.403681099,"pv":["E6","E3","E2","F2","B5","B6","D2","F5","F6"],"pvEdgeVisits":[89,57,30,27,5,3,3,2,1],"pvVisits":[89,57,30,27,5,6,3,2,1],"scoreLead":-0.841365265,"scoreMean":-0.841365265,"scoreSelfplay":-0.643457534,"scoreStdev":15.606272,"utility":-0.222693572,"utilityLcb":-0.447662167,"visits":89,"weight":88.87337987628432,"winrate":0.450582858},{"lcb":-1.11060376,"move":"B5","order":4,"prior":0.00439723395,"pv":["B5"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-4.95396185,"scoreMean":-4.95396185,"scoreSelfplay":-5.44648409,"scoreStdev":10.5671071,"utility":-0.870644001,"utilityLcb":-4.37064402,"visits":1,"weight":1.0,"winrate":0.139396243},{"lcb":-1.16454727,"move":"B4","order":5,"prior":0.000787151512,"pv":["B4"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-5.81449413,"scoreMean":-5.81449413,"scoreSelfplay":-6.96907425,"scoreStdev":9.32004386,"utility":-1.0120462,"utilityLcb":-4.51204622,"visits":1,"weight":1.0,"winrate":0.0854527354},{"lcb":-1.09901954,"move":"B6","order":6,"prior":0.000691668538,"pv":["B6"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-3.44325447,"scoreMean":-3.44325447,"scoreSelfplay":-5.49600983,"scoreStdev":12.4498331,"utility":-0.834932505,"utilityLcb":-4.33493252,"visits":1,"weight":1.0,"winrate":0.150980465}],"rootInfo":{"currentPlayer":"B","rawStScoreError":-1.0,"rawStWrError":-0.5,"rawVarTimeLeft":-1.0,"scoreLead":0.460134967,"scoreSelfplay":0.52757526,"scoreStdev":9.09794322,"symHash":"10155F672C0B5682E08EB339027AD849","thisHash":"E2B811FEE059E65F93697CB1DE35B543","utility":0.0432286293,"visits":1000,"weight":984.6275809289067,"winrate":0.563042759},"turnNumber":8}
: MoveNum: 14 HASH: B56076090A467FAA5A715507E32E1149
   A B C D E F G
 7 . . . . . . .
 6 . . X X2. . .
 5 . . X O O . .
 4 . O3O1X O . .
 3 . . . X O . .
 2 . . . X X O .
 1 . . . . . . .

koSIMPLEscoreAREAtaxNONEsui0whbNfpok1komi9
Root visits: 1000
New playouts: 1000
NN rows: 1680
NN batches: 1680
NN avg batch size: 1
PV: C3 E6 F1 B5 B6 D7 B3 G2 B7 F3 A4 G1 E1 G4 C7
Tree:
: T  20.45c W  24.32c S  -3.20c ( +0.5 L  +0.4) N    1000  --  C3 E6 F1 B5 B6 D7 B3
---Black(^)---
C3  : T  25.83c W  28.99c S  -2.83c ( +0.8 L  +0.7) LCB   18.42c P  8.84% WF 846.8 PSV     809 N     813  --  C3 E6 F1 B5 B6 D7 B3 G2
B2  : T   4.61c W   8.86c S  -3.74c ( +0.1 L  +0.2) LCB  -31.84c P 15.34% WF  72.4 PSV      38 N      83  --  B2 E6 B5 D7 C7 E7 F1 A6
F1  : T   1.59c W   8.49c S  -2.16c ( +0.4 L  +0.5) LCB  -27.90c P 14.65% WF  45.7 PSV      32 N      53  --  F1 E6 B5 B2 F3 F4 G2 D7
E6  : T -44.77c W -33.33c S  -9.20c ( -4.7 L  -4.8) LCB  -93.23c P 21.29% WF  14.6 PSV      16 N      23  --  E6 C3 C2 B2 F1
B3  : T -100.48c W -63.46c S -19.62c (-13.5 L -12.2) LCB -139.57c P 26.12% WF   6.9 PSV      11 N      14  --  B3 C3 C2 B2 B1 A3 B5
B5  : T -69.96c W -50.74c S -10.26c ( -5.8 L  -5.4) LCB -101.45c P 11.99% WF   5.5 PSV       6 N       9  --  B5 B2 C2 B3 F1 F3
F3  : T -58.21c W -48.57c S  -9.65c ( -4.4 L  -3.7) LCB -408.21c P  1.27% WF   0.8 PSV       0 N       1  --  F3
F6  : T -92.30c W -74.86c S -17.44c (-11.6 L  -9.7) LCB -442.30c P  0.17% WF   0.7 PSV       0 N       1  --  F6
G2  : T -68.13c W -56.41c S -11.72c ( -6.1 L  -5.0) LCB -418.13c P  0.11% WF   0.8 PSV       0 N       1  --  G2
C2  : T -75.65c W -63.34c S -12.31c ( -6.0 L  -5.0) LCB -425.65c P  0.06% WF   0.7 PSV       0 N       1  --  C2

: Response: {"id":"pvvisittest","isDuringSearch":false,"moveInfos":[{"lcb":0.618440806,"move":"C3","order":0,"prior":0.0883528665,"pv":["C3","E6","F1","B5","B6","D7","B3","G2","B7","F3","A4","G1","E1","G4","C7"],"pvEdgeVisits":[813,640,603,505,502,283,134,80,32,16,8,6,5,2,1],"pvVisits":[813,640,603,505,502,283,134,80,32,16,8,6,5,2,1],"scoreLead":0.696595134,"scoreMean":0.696595134,"scoreSelfplay":0.793653509,"scoreStdev":8.13732538,"utility":0.258338354,"utilityLcb":0.184167322,"visits":813,"weight":809.1635404432292,"winrate":0.64493046},{"lcb":0.414096181,"move":"B2","order":1,"prior":0.153434619,"pv":["B2","E6","B5","D7","C7","E7","F1","A6","A4","A5"],"pvEdgeVisits":[83,69,66,38,30,21,15,7,2,1],"pvVisits":[83,69,66,38,30,21,15,7,2,1],"scoreLead":0.165150088,"scoreMean":0.165150088,"scoreSelfplay":0.118007562,"scoreStdev":10.2437066,"utility":0.0460918691,"utilityLcb":-0.318409077,"visits":83,"weight":82.98491871335744,"winrate":0.54427509},{"lcb":0.437133047,"move":"F1","order":2,"prior":0.146502122,"pv":["F1","E6","B5","B2","F3","F4","G2","D7","B7","A6"],"pvEdgeVisits":[53,26,23,20,15,13,9,8,2,1],"pvVisits":[53,26,23,20,15,13,9,8,2,1],"scoreLead":0.53518269,"scoreMean":0.53518269,"scoreSelfplay":0.432143298,"scoreStdev":16.9535604,"utility":0.0159187997,"utilityLcb":-0.278954062,"visits":53,"weight":53.0,"winrate":0.542444783},{"lcb":0.160289506,"move":"E6","order":3,"prior":0.212886944,"pv":["E6","C3","C2","B2","F1"],"pvEdgeVisits":[23,10,4,3,1],"pvVisits":[23,10,4,3,1],"scoreLead":-4.84330971,"scoreMean":-4.84330971,"scoreSelfplay":-4.7290477,"scoreStdev":16.822838,"utility":-0.447713802,"utilityLcb":-0.932349144,"visits":23,"weight":22.743096393232474,"winrate":0.333373556},{"lcb":0.0430974992,"move":"B3","order":4,"prior":0.261169076,"pv":["B3","C3","C2","B2","B1","A3","B5"],"pvEdgeVisits":[14,13,8,7,3,2,1],"pvVisits":[14,13,8,7,3,2,1],"scoreLead":-12.1681985,"scoreMean":-12.1681985,"scoreSelfplay":-13.4926302,"scoreStdev":16.7550097,"utility":-1.00477179,"utilityLcb":-1.39566068,"visits":14,"weight":14.0,"winrate":0.182700673},{"lcb":0.133806866,"move":"B5","order":5,"prior":0.119903609,"pv":["B5","B2","C2","B3","F1","F3"],"pvEdgeVisits":[9,8,5,4,2,1],"pvVisits":[9,8,5,4,2,1],"scoreLead":-5.44694438,"scoreMean":-5.44694438,"scoreSelfplay":-5.77505655,"scoreStdev":17.5407855,"utility":-0.699551669,"utilityLcb":-1.01448005,"visits":9,"weight":9.0,"winrate":0.246281289},{"lcb":-0.992835081,"move":"F3","order":6,"prior":0.0126994969,"pv":["F3"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-3.74213624,"scoreMean":-3.74213624,"scoreSelfplay":-4.39467239,"scoreStdev":14.6443034,"utility":-0.582139054,"utilityLcb":-4.08213907,"visits":1,"weight":1.0,"winrate":0.257164925},{"lcb":-1.12430918,"move":"F6","order":7,"prior":0.0017052358,"pv":["F6"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-9.68535519,"scoreMean":-9.68535519,"scoreSelfplay":-11.6348562,"scoreStdev":17.0774297,"utility":-0.922978962,"utilityLcb":-4.42297898,"visits":1,"weight":1.0,"winrate":0.125690825},{"lcb":-1.03206649,"move":"G2","order":8,"prior":0.00111480814,"pv":["G2"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-5.01870155,"scoreMean":-5.01870155,"scoreSelfplay":-6.06047058,"scoreStdev":15.185776,"utility":-0.681311284,"utilityLcb":-4.1813113,"visits":1,"weight":1.0,"winrate":0.217933513},{"lcb":-1.06669333,"move":"C2","order":9,"prior":0.000553055143,"pv":["C2"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-4.95952082,"scoreMean":-4.95952082,"scoreSelfplay":-6.00005293,"scoreStdev":13.9286185,"utility":-0.756500945,"utilityLcb":-4.25650096,"visits":1,"weight":1.0,"winrate":0.183306672}],"rootInfo":{"currentPlayer":"B","rawStScoreError":-1.0,"rawStWrError":-0.5,"rawVarTimeLeft":-1.0,"scoreLead":0.377715317,"scoreSelfplay":0.44335226,"scoreStdev":9.31760314,"symHash":"0CAA03496D1088226717248B65E75E44","thisHash":"3F4A5C26F80BC4C79F44AC7833E26F3F","utility":0.207266493,"visits":1000,"weight":995.891555549819,"winrate":0.623544613},"turnNumber":14}
: tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: NN rows: 1680
: NN batches: 1680
: NN avg batch size: 1
: GPU -1 finishing, processed 1680 rows 1680 batches
: All cleaned up, quitting
