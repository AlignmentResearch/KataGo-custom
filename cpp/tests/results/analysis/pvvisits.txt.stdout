{"id":"pvvisittest","isDuringSearch":false,"moveInfos":[{"lcb":0.550073188,"move":"E3","order":0,"prior":0.0397898182,"pv":["E3","F4","F5","F6","F3","C4","B5","B4","B3","B6","C3","A5","E6","E7","G4","A3"],"pvEdgeVisits":[501,341,331,295,262,167,116,102,36,14,10,9,13,12,10,9],"pvVisits":[501,341,331,295,262,167,116,102,36,14,10,15,13,12,10,9],"scoreLead":0.727054998,"scoreMean":0.727054998,"scoreSelfplay":0.767866577,"scoreStdev":6.63931026,"utility":0.112668614,"utilityLcb":-0.00286120816,"visits":501,"weight":486.19663749964667,"winrate":0.591333839},{"lcb":0.505056348,"move":"F5","order":1,"prior":0.18750371,"pv":["F5","B5","C4","B4","B3","E3","D2","E2","F2","B2","E1","C2","F3"],"pvEdgeVisits":[260,227,134,94,84,73,63,39,16,9,4,2,1],"pvVisits":[260,227,134,94,84,73,63,39,16,9,4,2,1],"scoreLead":0.576046395,"scoreMean":0.576046395,"scoreSelfplay":0.704911421,"scoreStdev":11.1433361,"utility":0.0392741566,"utilityLcb":-0.14247914,"visits":260,"weight":259.5575635529757,"winrate":0.569968239},{"lcb":0.415005721,"move":"F4","order":2,"prior":0.361343056,"pv":["F4","C4","C3","B5","B4","F5","E3","E6","C5","E4","B6","C4","E5"],"pvEdgeVisits":[146,121,108,103,78,32,19,16,6,5,3,2,1],"pvVisits":[146,121,108,103,78,32,19,16,6,5,3,2,1],"scoreLead":-0.119258103,"scoreMean":-0.119258103,"scoreSelfplay":-0.125941616,"scoreStdev":9.67756693,"utility":-0.111081208,"utilityLcb":-0.321783473,"visits":146,"weight":146.0,"winrate":0.49025653},{"lcb":0.370236931,"move":"E6","order":3,"prior":0.403681099,"pv":["E6","E3","E2","F2","B5","B6","D2","F5","F6"],"pvEdgeVisits":[89,57,30,27,5,3,3,2,1],"pvVisits":[89,57,30,27,5,6,3,2,1],"scoreLead":-0.841365265,"scoreMean":-0.841365265,"scoreSelfplay":-0.643457534,"scoreStdev":15.606272,"utility":-0.222693572,"utilityLcb":-0.447662167,"visits":89,"weight":88.87337987628432,"winrate":0.450582858},{"lcb":-1.11060376,"move":"B5","order":4,"prior":0.00439723395,"pv":["B5"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-4.95396185,"scoreMean":-4.95396185,"scoreSelfplay":-5.44648409,"scoreStdev":10.5671071,"utility":-0.870644001,"utilityLcb":-4.37064402,"visits":1,"weight":1.0,"winrate":0.139396243},{"lcb":-1.16454727,"move":"B4","order":5,"prior":0.000787151512,"pv":["B4"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-5.81449413,"scoreMean":-5.81449413,"scoreSelfplay":-6.96907425,"scoreStdev":9.32004386,"utility":-1.0120462,"utilityLcb":-4.51204622,"visits":1,"weight":1.0,"winrate":0.0854527354},{"lcb":-1.09901954,"move":"B6","order":6,"prior":0.000691668538,"pv":["B6"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-3.44325447,"scoreMean":-3.44325447,"scoreSelfplay":-5.49600983,"scoreStdev":12.4498331,"utility":-0.834932505,"utilityLcb":-4.33493252,"visits":1,"weight":1.0,"winrate":0.150980465}],"rootInfo":{"currentPlayer":"B","rawStScoreError":-1.0,"rawStWrError":-0.5,"rawVarTimeLeft":-1.0,"scoreLead":0.460134967,"scoreSelfplay":0.52757526,"scoreStdev":9.09794322,"symHash":"10155F672C0B5682E08EB339027AD849","thisHash":"E2B811FEE059E65F93697CB1DE35B543","utility":0.0432286293,"visits":1000,"weight":984.6275809289067,"winrate":0.563042759},"turnNumber":8}
{"id":"pvvisittest","isDuringSearch":false,"moveInfos":[{"lcb":0.618440806,"move":"C3","order":0,"prior":0.0883528665,"pv":["C3","E6","F1","B5","B6","D7","B3","G2","B7","F3","A4","G1","E1","G4","C7"],"pvEdgeVisits":[813,640,603,505,502,283,134,80,32,16,8,6,5,2,1],"pvVisits":[813,640,603,505,502,283,134,80,32,16,8,6,5,2,1],"scoreLead":0.696595134,"scoreMean":0.696595134,"scoreSelfplay":0.793653509,"scoreStdev":8.13732538,"utility":0.258338354,"utilityLcb":0.184167322,"visits":813,"weight":809.1635404432292,"winrate":0.64493046},{"lcb":0.414096181,"move":"B2","order":1,"prior":0.153434619,"pv":["B2","E6","B5","D7","C7","E7","F1","A6","A4","A5"],"pvEdgeVisits":[83,69,66,38,30,21,15,7,2,1],"pvVisits":[83,69,66,38,30,21,15,7,2,1],"scoreLead":0.165150088,"scoreMean":0.165150088,"scoreSelfplay":0.118007562,"scoreStdev":10.2437066,"utility":0.0460918691,"utilityLcb":-0.318409077,"visits":83,"weight":82.98491871335744,"winrate":0.54427509},{"lcb":0.437133047,"move":"F1","order":2,"prior":0.146502122,"pv":["F1","E6","B5","B2","F3","F4","G2","D7","B7","A6"],"pvEdgeVisits":[53,26,23,20,15,13,9,8,2,1],"pvVisits":[53,26,23,20,15,13,9,8,2,1],"scoreLead":0.53518269,"scoreMean":0.53518269,"scoreSelfplay":0.432143298,"scoreStdev":16.9535604,"utility":0.0159187997,"utilityLcb":-0.278954062,"visits":53,"weight":53.0,"winrate":0.542444783},{"lcb":0.160289506,"move":"E6","order":3,"prior":0.212886944,"pv":["E6","C3","C2","B2","F1"],"pvEdgeVisits":[23,10,4,3,1],"pvVisits":[23,10,4,3,1],"scoreLead":-4.84330971,"scoreMean":-4.84330971,"scoreSelfplay":-4.7290477,"scoreStdev":16.822838,"utility":-0.447713802,"utilityLcb":-0.932349144,"visits":23,"weight":22.743096393232474,"winrate":0.333373556},{"lcb":0.0430974992,"move":"B3","order":4,"prior":0.261169076,"pv":["B3","C3","C2","B2","B1","A3","B5"],"pvEdgeVisits":[14,13,8,7,3,2,1],"pvVisits":[14,13,8,7,3,2,1],"scoreLead":-12.1681985,"scoreMean":-12.1681985,"scoreSelfplay":-13.4926302,"scoreStdev":16.7550097,"utility":-1.00477179,"utilityLcb":-1.39566068,"visits":14,"weight":14.0,"winrate":0.182700673},{"lcb":0.133806866,"move":"B5","order":5,"prior":0.119903609,"pv":["B5","B2","C2","B3","F1","F3"],"pvEdgeVisits":[9,8,5,4,2,1],"pvVisits":[9,8,5,4,2,1],"scoreLead":-5.44694438,"scoreMean":-5.44694438,"scoreSelfplay":-5.77505655,"scoreStdev":17.5407855,"utility":-0.699551669,"utilityLcb":-1.01448005,"visits":9,"weight":9.0,"winrate":0.246281289},{"lcb":-0.992835081,"move":"F3","order":6,"prior":0.0126994969,"pv":["F3"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-3.74213624,"scoreMean":-3.74213624,"scoreSelfplay":-4.39467239,"scoreStdev":14.6443034,"utility":-0.582139054,"utilityLcb":-4.08213907,"visits":1,"weight":1.0,"winrate":0.257164925},{"lcb":-1.12430918,"move":"F6","order":7,"prior":0.0017052358,"pv":["F6"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-9.68535519,"scoreMean":-9.68535519,"scoreSelfplay":-11.6348562,"scoreStdev":17.0774297,"utility":-0.922978962,"utilityLcb":-4.42297898,"visits":1,"weight":1.0,"winrate":0.125690825},{"lcb":-1.03206649,"move":"G2","order":8,"prior":0.00111480814,"pv":["G2"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-5.01870155,"scoreMean":-5.01870155,"scoreSelfplay":-6.06047058,"scoreStdev":15.185776,"utility":-0.681311284,"utilityLcb":-4.1813113,"visits":1,"weight":1.0,"winrate":0.217933513},{"lcb":-1.06669333,"move":"C2","order":9,"prior":0.000553055143,"pv":["C2"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-4.95952082,"scoreMean":-4.95952082,"scoreSelfplay":-6.00005293,"scoreStdev":13.9286185,"utility":-0.756500945,"utilityLcb":-4.25650096,"visits":1,"weight":1.0,"winrate":0.183306672}],"rootInfo":{"currentPlayer":"B","rawStScoreError":-1.0,"rawStWrError":-0.5,"rawVarTimeLeft":-1.0,"scoreLead":0.377715317,"scoreSelfplay":0.44335226,"scoreStdev":9.31760314,"symHash":"0CAA03496D1088226717248B65E75E44","thisHash":"3F4A5C26F80BC4C79F44AC7833E26F3F","utility":0.207266493,"visits":1000,"weight":995.891555549819,"winrate":0.623544613},"turnNumber":14}
